### PR TITLE
Display military points on the left side of the wonder instead of over it

### LINKED ISSUE #65: Display military points next to the board instead of over it

### PR DESCRIPTION
**Pull request by ArchangelX360**
Resolves #65

image
image

### CODE REVIEW

**Comment by joffrey-bion** [comment]
Did you change the defeat token ?

File: sw-ui/src/main/kotlin/org/luxons/sevenwonders/ui/components/game/Board.kt [hunk 1]
```diff
@@ -126,20 +126,24 @@ private fun RBuilder.wonderComponent(wonder: ApiWonder, military: Military) {
...
             }
-            victoryPoints(military.victoryPoints) {
+            styledDiv {
                 css {
                     position = Position.absolute
-                    top = 25.pct // below the wonder name
-                    left = 60.pct
-                    zIndex = 2 // go above the wonder, but below the table cards
+                    top = 20.pct // below the wonder name
+                    right = -(80.px)
```
[the remark this answers is not in the document]
Comment (ArchangelX360) [reply sw-ui/src/main/kotlin/org/luxons/sevenwonders/ui/components/game/Board.kt]: Done.

**Comment by ArchangelX360** [comment]
> Did you change the defeat token ?

Yes, for it to have the same `width` than the victory one, saving some CSS... 😄

**Comment by joffrey-bion** [comment]
> > Did you change the defeat token ?
> 
> Yes, for it to have the same `width` than the victory one, saving some CSS... 😄

It could be used in multiple different contexts (e.g. the board summary), so that won't work everywhere.
If we need it bigger in some places, we will have lost some resolution

**Pull request merged**